project/MicroService.scala

```diff
@@ -23,10 +22,21 @@ trait MicroService {
   lazy val plugins : Seq[Plugins] = Seq.empty
   lazy val playSettings : Seq[Setting[_]] = Seq.empty
 
+  lazy val scoverageSettings = {
+    import scoverage.ScoverageKeys
+    Seq(
+      // Semicolon-separated list of regexs matching classes to exclude
+      ScoverageKeys.coverageExcludedPackages := """uk\.gov\.hmrc\.[a-zA-Z0-1]*;com\.kenshoo\.play\..*;.*\.Routes.*;.*\.Reverse.*;.*\.javascript\..*""",
```

Comment:
It should be """uk\.gov\.hmrc\.BuildInfo;com\.kenshoo\.play\..;.\.Routes.;.\.Reverse.;.\.javascript\..*"""